src/components/JUPTable/JUPTable.tsx

```diff
@@ -0,0 +1,184 @@
+import React from "react";
+import {
+  Box,
+  Paper,
+  styled,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TablePagination,
+  TableRow,
+  TableSortLabel,
+  Typography,
+} from "@mui/material";
+import SLink from "components/SLink";
+import { Data, IHeadCellProps } from "views/Dashboard/components/Widgets/TransactionsWidget/TransactionsWidget";
+import { ITransaction } from "types/NXTAPI";
+import { visuallyHidden } from "@mui/utils";
+import { TableRowsPerPageOptions } from "utils/common/constants";
+
+// might still want to use this concept
+// function createWidgetRow(date: string, qty: number, toFrom: string): Data {
+//   return {
+//     date,
+//     qty,
+//     toFrom,
+//   };
+// }
+
+interface ITableTitleProps {
+  title: string;
+}
+const TableTitle: React.FC<ITableTitleProps> = ({ title }) => {
+  return (
+    <>
+      {
+        <TitleText variant="h6" id="tableTitle">
+          <SLink href="/transactions">{title}</SLink>
```

Comment:
if this is supposed to be a generic table component, then is quite odd to see a specific url here.

Reply:
Good catch, I noticed this in a different branch and fixed it a while back. Fix is in the WIP blocks widget branch which I expect to merge into master prior to needing to re-use this new component.